taskcluster/ci/build/kind.yml

```diff
@@ -5,6 +5,7 @@
 loader: taskgraph.loader.transform:loader
 
 transforms:
+    - mozillavpn_taskgraph.transforms.release_index:transforms
```

Comment:
Open question, do we want to add both the unsigned and signed build tasks to this index? I was kind of thinking we'd only index the signed tasks, but maybe there's benefit to also adding the build tasks.

Reply:
Oh, I think you're doing this just because there are no signed Windows builds yet. If this is temporary, then that makes sense to me!

Reply:
Also if you move this transform below the `build:transforms`, you wouldn't need to add `routes` to the build schema. That said, I propose a better fix below.

Reply:
Yes, i've only added this to have the windows.msi also there, for now :)